GraphicsTools macOS target does not build

```
/Users/brianheim/git/dn-m/GraphicsTools/GraphicsToolsTests/GlyphTests.swift:38:25: error: use of unresolved identifier 'UIBezierPath'
        let xLinePath = UIBezierPath()
                        ^~~~~~~~~~~~
AppKit.NSBezierPath:1:12: note: did you mean 'NSBezierPath'?
open class NSBezierPath : NSObject, NSCopying, NSCoding {
           ^
/Users/brianheim/git/dn-m/GraphicsTools/GraphicsToolsTests/GlyphTests.swift:46:25: error: use of unresolved identifier 'UIBezierPath'
        let yLinePath = UIBezierPath()
                        ^~~~~~~~~~~~
AppKit.NSBezierPath:1:12: note: did you mean 'NSBezierPath'?
open class NSBezierPath : NSObject, NSCopying, NSCoding {
           ^
<unknown>:0: warning: 'cacheParamsComputed' is deprecated
<unknown>:0: warning: 'cacheAlphaComputed' is deprecated
<unknown>:0: warning: 'keepCacheWindow' is deprecated
<unknown>:0: error: 'memoryless' is unavailable
Metal.MTLCommandBufferError:55:14: note: 'memoryless' has been explicitly marked unavailable here
        case memoryless
             ^
```

These errors show up while trying to build the test target. Is this intended to be built for macOS?

--- a/GraphicsToolsTests/GlyphTests.swift
+++ b/GraphicsToolsTests/GlyphTests.swift
@@ -26,7 +26,8 @@ class GlyphTests: XCTestCase {
         let layer = CAShapeLayer(glyph)
         layer.renderToPDF(name: "square_glyph")
     }
-    
+
+    #if os(iOS)
     func testSquarePositionedGlyph() {
         let square = Path.rectangle(x: 0, y: 0, width: 10, height: 10)
         let glyph = Glyph(path: square, position: Point(x: 30, y: 70))
@@ -58,6 +59,7 @@ class GlyphTests: XCTestCase {
         container.addSublayer(layer)
         container.renderToPDF(name: "square_positioned")
     }
+    #endif
 
     func testSquare() {
         let square = Path.square(center: Point(x: 20, y: 20), width: 40)

